requirements-build.txt

```diff
@@ -1,3 +1,2 @@
 Babel==2.9.1
 PyInstaller==5.3
-setuptools<45.0.0
```

Comment:
This got already removed on master, I think rebasing this branch against master would solve that.

Reply:
You've approved and commented so I'm confused

Should I do it now or is it just a tip for the future PRs?

Reply:
Done :)